### PR TITLE
update install repos from last workshop 2015 LN

### PR DESCRIPTION
**Pull request by MalloZup**
I think, that in this way we can avoid some mistake with wrongs repos and it will be more clear.

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -31,10 +31,19 @@ repository. Installation on openSUSE is therefore pretty simple:
...
-zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
-zypper in openQA
+
+- 13.2
+  zypper ar -f obs://devel:openQA/openSUSE_13.2 openQA
+  zypper ar -f obs://devel:openQA:13.2/openSUSE_13.2 openQA-perl-modules
+
+- Leap 42.1
+  zypper ar -f obs://devel:openQA/openSUSE_Leap_42.1 openQA
+  zypper ar -f obs://devel:openQA:42/openSUSE_Leap_42.1 openQA-perl-modules
```
Comment (sysrich) [inline docs/Installing.asciidoc]: Paging @lnussel - am I mistaken, or did we make the devel:openQA:42 repo obsolete by putting the required modules in Leap:42.1?

Comment (nilxam) [inline docs/Installing.asciidoc]: devel:openQA should fullfilled requirements. so far the only one missed perl module after Leap 42.1 released is perl-Critic ie. it's the only one package in devel:openQA:42 now, but it's a special case - do not need it to run openQA, unless you want to run cpanfile or make check. BTW perl-Critic does not build in devel:openQA:42 now...

Comment (coolo) [inline docs/Installing.asciidoc]: I would still leave it in the instructions. it's just a matter of time that openQA needs something not in leap

Comment (lnussel) [inline docs/Installing.asciidoc]: but  then we should name the project :42.1 :)

**Comment by lnussel** [comment]
done. we have devel:openQA:Leap:42.1 now

**Comment by aaannz** [comment]
@MalloZup Can you update this as discussed?
`zypper ar -f obs://devel:openQA:Leap:42.1/openSUSE_Leap_42.1 openQA-perl-modules`

**Comment by MalloZup** [comment]
aananz i deleted the repo, so could you guys add for me?

**Comment by MalloZup** [comment]
ok merged i close

**Pull request closed without merging**